package.json

```diff
@@ -46,10 +110,16 @@
     "url": "https://github.com/semantic-release/condition-travis.git"
   },
   "scripts": {
-    "coverage": "tap --coverage-report=lcov",
-    "coverage:upload": "tap --coverage-report=text-lcov | coveralls",
-    "pretest": "standard",
+    "clean": "rimraf coverage && rimraf .nyc_output",
+    "cm": "git-cz",
+    "commitmsg": "commitlint -e",
+    "coverage": "nyc check-coverage",
+    "coveralls": "nyc report --reporter=text-lcov | coveralls",
+    "lint": "eslint lib test",
+    "precommit": "npm run lint",
+    "prepush": "ava -v",
```

Comment:
can you remove husky and the commit hooks? Things won’t get merged until CI passes and at least while working on pull requests there are valid valid cases to have both invalid commit messages and failing tests.

Reply:
Ok. That mean I should remove `commitlint` as well, because it's currently used only on the `commitmsg` hook ?

That would leave us with no commit format validation because unfortunately there is no easy way to tell `commitlint` to check all the commit within a PR. `commitlint` has a `from` argument but we would need to figure out what is the first commit in the PR.
On the other hand validating only on git hook is not safe either anyway.

Reply:
yes, let’s remove all the git hooks for now, thanks 👍

Reply:
Ok, it's done.